python/foxglove-sdk-examples/ws-param-server/main.py

```diff
@@ -0,0 +1,97 @@
+"""
+This implements a parameter server for live visualization.
+
+View and edit parameters from a Parameters panel in Foxglove:
+https://docs.foxglove.dev/docs/visualization/panels/parameters
+"""
+
+import logging
+from typing import Optional
+import foxglove
+import time
+
+
+from foxglove import Capability
+from foxglove import ParameterType, ParameterValue, Parameter
+
+
+class ParameterStore(foxglove.ServerListener):
+    def __init__(self, parameters: list[Parameter]) -> None:
+        self.parameters = parameters
+
+    # Foxglove server callback
+    def on_get_parameters(
+        self,
+        client: foxglove.Client,
+        param_names: list[str],
+        request_id: Optional[str] = None,
+    ) -> list[Parameter]:
+        logging.debug(f"on_get_parameters: {param_names}, {client.id}, {request_id}")
+        return self.parameters
```

Comment:
This is ignoring the param names, that seems like the wrong thing to do in an example

Reply:
oh yes, good catch